frontend/www/js/omegaup/components/problem/CollectionProblem.vue

```diff
@@ -14,9 +14,7 @@
             {{ title }}
           </h6>
           <slot name="problem-count"></slot>
-          <a class="btn btn-primary" :href="href">{{
-            T.problemcollectionViewProblems
-          }}</a>
+          <slot name="collection-button"></slot>
```

Comment:
```suggestion
          <slot name="button"></slot>
```

dado que este componente es una colección, no hay necesidad de mencionar que este es un botón de la colección.